app/controllers/homepage_controller.rb

```diff
@@ -0,0 +1,10 @@
+class HomepageController < ApplicationController
+  def index
+    @works = Work.all
+    @movies = Work.all.select { |w| w.category == "movie" }
```

Comment:
You should have limited to 10 works sorted by votes.  

For reference you can do that like this:

```suggestion
    @movies = Work.where(category: "movie" ).order(votes: :desc).limit(10)
```

It's probably too late of feedback for you to apply to Video Store API but might be useful to know.